charts/home-assistant/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: 0.113.3
+appVersion: 0.114.0
 description: Home Assistant
 name: home-assistant
-version: 1.1.0
+version: 1.1.1
```

Comment:
Shouldn't this bump to 1.2.0 as this isn't really a bugfix but actually introduces a change to the chart?

Reply:
I'm happy either way to be honest. I've been bumping the HAAS version and incrementing the chart as a patch version.

Reply:
Personally, I've been bumping only the patch version as well whenever updating any chart's image version.

I suggest that we reserve minor semver changes (e.g. going from 1.1.0 to 1.2.0) for functional changes to the _chart_ itself and not the underlying application changes.